device-types/HPE/Aruba-IAP-505H.yaml

```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: HPE
+model: Aruba IAP-505H
+slug: hpe-aruba-iap-505h
+part_number: R3V48A
+u_height: 0.0
+is_full_depth: false
+airflow: passive
+comments: '[Aruba 500H Series Datasheet](https://www.arubanetworks.com/assets/ds/DS_AP500HSeries.pdf)'
+weight: 360.0
+weight_unit: g
+console-ports:
+  - name: USB 2.0
+    type: usb-a
+    label: USBA
+power-ports:
+  - name: DC Power Interface
+    type: dc-terminal
+    label: DC Power
+interfaces:
+  - name: BLUETOOTH / Zigbee
```

Comment:
Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one

Reply:
I changed it according to the Data Sheet.